src/api/spec/controllers/webui/obs_factory/staging_projects_controller_spec.rb

```diff
@@ -121,33 +136,13 @@
   end
 
   describe 'GET #show' do
-    let(:source_package) { create(:package) }
-    let(:target_package) { create(:package, name: 'target_package', project: factory) }
-    let(:bs_request) do
-      create(:bs_request_with_submit_action,
-             target_project: factory.name,
-             target_package: target_package.name,
-             source_project: source_package.project.name,
-             source_package: source_package.name)
-    end
-    let(:description) do
-      <<-DESCRIPTION
-        requests:
-          - { id: #{bs_request.number} }
-      DESCRIPTION
-    end
-
     context 'with a existent factory_staging_project' do
-      before do
-        bs_request.update(state: 'declined')
-        factory_staging_a.update(description: description)
-      end
-
       context 'requesting html' do
-        subject { get :show, params: { project: factory, project_name: 'A' } }
+        subject! { get :show, params: { project: factory, project_name: 'A' } }
 
         it { is_expected.to have_http_status(:success) }
         it { expect(subject).to render_template(:show) }
```

Comment:
While we are here

`it { is_expected.to render_template(:show) }`

Reply:
I'll change it. But please note that this was already there. I can not refactor every file I touch:wink: Even though I already try.